conf/machine/phygate-tauri-l-imx8mm-2.extra.conf

```diff
@@ -1,5 +1,8 @@
 require conf/machine/include/phytec-imx8mm.inc
 
+# adu compatibility id
+ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "1"
```

Comment:
that's wrong; only for kirkstone-next must be set to 1;

Reply:
when we integrate this PR in kirkstone-next, there it will be set to 2. reason: this PR is incopatible with
tauri with ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "0"

Reply:
or do you mean, i have to set it to 2 here and in kirkstone-next later to 3?

Reply:
why are rpi and tauri images now incompatible?

Reply:
reason: updates u-boot variable names?